test/local-fs.spec.ts

```diff
@@ -114,6 +115,14 @@ describe(`the local filesystem implementation`, () => {
                 .then(() => expect(fs.loadTextFile(fileName)).to.eventually.equals(newContent));
         });
 
+        it(`ignores events from ignored dir in watcher`, () => {
```

Comment:
If the file system is 100% in sync then there is no justification for this addition to the API. please remove it.